src/encoded/static/components/region_search.js

```diff
@@ -189,6 +198,10 @@ class AdvSearch extends React.Component {
         const id = url.parse(this.context.location_href, true);
         const region = id.query.region || '';
 
+        if (this.state.genome === '') {
+            this.state.genome = context.assembly ? context.assembly : regionGenomes[0].value;
```

Comment:
React’s `this.state` is immutable through any mechanism besides React’s `setState()`. Did `setState()` not work here? Calling `setState()` within `render()` is really tricky because that can cause infinite render loops, but I think it should work here because of the `if` statement that gates it.